CONTRIBUTING.md

```diff
@@ -1,92 +1,155 @@
-## Pull Request Process
+As with most projects, prior to starting to code on a bug fix or feature request, please post in the respective GitHub issue saying you want to volunteer, and then wait for a positive response. And if there is no issue for it yet, create it first.
 
-1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
-   build.
-2. Update the README.md with details of changes to the interface, this includes new environment 
-   variables, exposed ports, useful file locations and container parameters.
-3. Increase the version numbers in any examples files and the README.md to the new version that this
-   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
-4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
-   do not have permission to do that, you may request the second reviewer to merge it for you.
+This helps make sure:
+1. Two people aren't working on the same thing.
+2. This is something Aim's maintainers believe should be implemented/fixed.
+3. Any API, UI, or deeper architectural changes that need to be implemented have been fully thought through by the community together with Aim's maintainers.
 
-## Code of Conduct
+Please follow [Aim Contributor Covenant Code of Conduct](./CODE_OF_CONDUCT.md) in all your interactions with the project.
 
-### Our Pledge
+## Governance
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
+This section describes governance processes we follow in developing Aim.
 
-### Our Standards
+### Persons of Interest
 
-Examples of behavior that contributes to creating a positive environment
-include:
+#### Authors
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+- Gevorg Soghomonyan ([SGevorg](https://github.com/SGevorg))
+- Gor Arakelyan ([gorarakelyan](https://github.com/gorarakelyan))
 
-Examples of unacceptable behavior by participants include:
+#### Core Maintainers
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
-advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic
-  address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+- [alberttorosyan](https://github.com/alberttorosyan)
```

Comment:
I suggest the following structure:
**Authors**
Gor, Gev

**Primary maintainers**
Ruben K, Albert, mahnerak

**Core Maintainers**
Aram
Arsen
Arthur
Hamik
Karo
Mihran
Ruben A.
Vardges

Reply:
Core contributors vs Core maintainers @gorarakelyan thoughts?

Reply:
sounds good